bayesian/api/api_v2.py

```diff
@@ -202,6 +206,64 @@ def get(ecosystem, package, version):
 
         raise HTTPError(404, msg)
 
+    @staticmethod
+    def post():
+        """Handle the POST REST API call.
+
+        Component Analyses Batch is 4 Step Process:
+        1. Gather and clean Request.
+        2. Query GraphDB.
+        3. Build Stack Recommendation and Build Unknown Packages and Trigger componentApiFlow.
+        4. Handle Unknown Packages and Trigger bayesianApiFlow.
+        """
+        response_template: Tuple = namedtuple("response_template", ["message", "status"])
+        ingestion_disabled_msg: str = "No data found for any package in manifest file. " \
+                                      "Ingestion flow skipped as DISABLE_UNKNOWN_PACKAGE_FLOW " \
+                                      "is enabled"
+        no_package_available_msg: str = "No Package in given manifest is available. " \
+                                        "Packages will be available shortly, " \
+                                        "Please retry after some time."
+
+        # Step1: Gather and clean Request
+        input_json: Dict = request.get_json()
+        ecosystem: str = input_json.get('ecosystem')
+        try:
+            packages_list, normalised_input_pkgs = ca_validate_input(input_json, ecosystem)
```

Comment:
You can move all the code under one try and multiple catch.  That will improve readability of the code.

Reply:
Ack!